src/scripts/main.js

```diff
@@ -2,7 +2,24 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dash = document.querySelector('.dashboard');
```

Comment:
I'd recommend to add descriptive names
```suggestion
const dashboardRef = document.querySelector('.dashboard');
```